src/modal/modal.ts

```diff
@@ -21,7 +21,7 @@ export interface NgbModalOptions {
   /**
    * Size of a new modal window.
    */
-  size?: 'sm' | 'lg';
+  size?: 'sm' | 'lg' | 'md';
```

Comment:
It doesn't matter technically, but I would find it clearer in the documentation if the sizes were sorted: 

    'sm' | 'md' | 'lg'

But most importantly, I don't see the 'md' size documented anywhere in bootstrap. [The documentation](https://v4-alpha.getbootstrap.com/components/modal/#optional-sizes) says:

> Modals have two optional sizes, available via modifier classes to be placed on a .modal-dialog

Does the md size really exist?